unitTests/ui/FacetSearchTest.ts

```diff
@@ -144,23 +153,94 @@ export function FacetSearchTest() {
 
           it('arrow navigation', function(done) {
             searchPromise.then(() => {
-              expect($$($$(facetSearch.searchResults).findAll('li')[0]).hasClass('coveo-facet-search-current-result')).toBe(true);
+              expect(getSearchResult(0).hasClass('coveo-facet-search-current-result')).toBe(true);
```

Comment:
Ouff, that test was/is pretty long and hard to follow. 

Could have taken the time to maybe refactor it a bit here ? I guess it's simply trying to check `down / up / looping around` all inside the same test, which could be simply exploded a bit.